wayland/getting-started.txt

```diff
@@ -0,0 +1,292 @@
+a terminal, and dmenu, Wayland can work quite well for you.
+
+
+Index
+________________________________________________________________________________
+
+- Getting Started                                                          [0.0]
+    - Caveats                                                              [0.1]
+    - Benefits                                                             [0.2]
+    - Prerequisites                                                        [0.3]
+
+- Using Xwayland                                                           [1.0]
+
+- Xorgless                                                                 [2.0]
+    - Alternatives                                                         [2.1]
+
+- Post Installation                                                        [3.0]
+- References                                                               [4.0]
+
+
+[0.0] Getting Started
+________________________________________________________________________________
+
+Because Wayland is not officially supported, all Wayland packages must be
+managed by the users themselves. However, most of the software required for
+Wayland and most compositors can be found in the main repositories, and Wayland
+support can be added by simply modifying the build scripts. 
+
+Several user repositories already exist which provide Wayland packages. 
+
+* For a repository which supports Xwayland with sway and firefox, check out
+  $/Himmalerin/kiss-wayland
+
+* For a repository that opts for a strictly xorgless approach and uses hikari
+  and chromium, look at $/dilyn-corner/KISS-me
+
+
+    [0.1] Caveats
+    ____________________________________________________________________________
+    
+    While the Wayland protocol is more or less done, the compositors available
+    can differ in protocol implementation. Xorg is still in wide use and 
+    alternatives to usual software must be sought. Luckily, many Xorg analogues
+    already exist for Wayland - see [2.1].
+    
+    The major downside to Wayland is the reduction in choice. While the Wayland
+    protocol does not define what a compositor shall depend on, they can
+    potentially require udev, PAM, and elogind. Additionally, Xorg has nearly an
+    unlimited number of window managers and desktop environments to select from.
+    With Wayland, there are far fewer compositors to choose.
+    
+    wlroots provides a standard that many Wayland compositors choose to 
+    implement, and most software can be expected to work across most wlroots
+    compositors. For those which don't utilize wlroots, however, finding
+    software which works can be trickier.
+    
+    In short, the general limitations users can expect are as follows:
+    
+    - Limited NVIDIA support [8]
+    - Nonstandard and compositor specific software [9]
+    - No Wayland support in SDL 1 (must use XWayland) [10]
+    - Starting compositors without logind requires SUID permissions [11]
```

Comment:
https://github.com/swaywm/wlroots/pull/2393
https://github.com/torvalds/linux/commit/45bc3d26c95a

Reply:
Now THIS is exciting news I can't believe I missed.